.gitignore

```diff
@@ -1,3 +1,4 @@
 fonts/.DS_Store
 Img/.DS_Store
-.DS_Store
\ No newline at end of file
+.DS_Store
+.prettierrc
```

Comment:
.prettierrc doesn't have to be ignored, unless you wish so :)

Reply:
ok